wpilibjIntegrationTests/build.gradle

```diff
@@ -24,8 +24,11 @@ dependencies {
     compile 'com.googlecode.junit-toolbox:junit-toolbox:2.0'
     compile 'org.apache.ant:ant:1.9.4'
     compile 'org.apache.ant:ant-junit:1.9.4'
+    compile files(wpilibj.sourceSets.test.output.classesDir)
 }
 
+compileJava.dependsOn tasks.getByPath(':wpilibj:testClasses')
```

Comment:
Ok, so wpilibj:testClasses refers to files in `test` sourceSet for wpilibj? I don't think that exists. I believe you that this is right I just want to understand it.